docs/en/stack/ml/anomaly-detection/index.asciidoc

```diff
@@ -10,6 +10,8 @@ include::ml-api-quickref.asciidoc[leveloffset=+1]
 
 include::anomaly-examples.asciidoc[leveloffset=+1]
 
+include::../get-started/index.asciidoc[leveloffset=+2]
```

Comment:
I'm not sure that the Tutorial should be a subsection of Examples. What do you think about publishing it in the first level of the Anomaly detection book?

Reply:
IMO it is less important now that the "Finding anomalies" is our intended end-to-end description. So if not in the examples, I'd likely embed it under that page.  For now, I'll leave in examples, since I still need to revisit the location of all the items in that section.